src/v2/Apps/_Preferences2/PreferencesApp.tsx

```diff
@@ -1,9 +1,176 @@
-import { Text } from "@artsy/palette"
+import {
+  Box,
+  Button,
+  Checkbox,
+  Column,
+  Flex,
+  GridColumns,
+  Separator,
+  Spacer,
+  Text,
+} from "@artsy/palette"
+import { useState } from "react"
 
 export const PreferencesApp: React.FC = () => {
+  const [uncheckedAll, setUncheckedAll] = useState(false)
+
   return (
     <>
-      <Text variant="xl">Preferences App</Text>
+      <Text variant="xl" mt={4} mb={6}>
+        Preferences Center
+      </Text>
+
+      <Separator mt={2} />
+
+      <GridColumns>
+        <Column
+          span={12}
+          pr={12}
+          mt={2}
+          display="flex"
+          justifyContent="space-between"
```

Comment:
All these props can be deleted; just use `Column` alone with the `span` prop.

Reply:
Updated.